### ISSUE TITLE
[index] inline index roles currently issue warning

### ISSUE THREAD

**Issue by mmcky** [executablebooks/sphinx-tomyst#33]
Currently the `inline` index roles issue a warning. 

This might be able to be implemented by iterating on `node.parent.children` for each `inline index` to recover the elements of the `inline` markup.

**Comment by mmcky** [comment]
This is the `xml` for the third example in `tests/docutils/roles.rst`

```rst
This is pair of entries :index:`index entries <pair: index; entry>`
```

```xml
<paragraph>
    This is pair of entries 
    <index entries="('pair',\ 'index;\ entry',\ 'index-2',\ '',\ None)">
    <target ids="index-2">
    index entries
```

**Comment by mmcky** [comment]
@chrisjsewell does `myst have equivalent support for this rst syntax for `roles`?

**Comment by mmcky** [comment]
This is now fixed in #80
